lib/src/dartdoc_analyzer.dart

```diff
@@ -28,3 +32,43 @@ Suggestion getDartdocRunFailedSuggestion([DartdocResult result]) {
     score: 10.0,
   );
 }
+
+/// Creates a report section about documentation coverage.
+/// 20% coverage grants the maximum number of points.
+ReportSection documentationCoverageSection({
+  @required int documented,
+  @required int total,
+}) {
+  final maxPoints = 10;
+  final ratio = total <= 0 ? 1.0 : documented / total;
+  final accepted = ratio >= 0.2;
+  final percent = ratio.toStringAsFixed(1);
+  final summary = StringBuffer();
+  summary.write('$documented out of $total ($percent %) '
```

Comment:
I would still like the first paragraph to give the rule. (`*10 points*: 20% or more of the public api has dartdoc.` or similar) Then the following paragraphs can give details

Reply:
Done. (also for the example check)